packages/components/src/VirtualizedList/RowCollapsiblePanel/__snapshots__/RowCollapsiblePanel.test.js.snap

```diff
@@ -0,0 +1,91 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`RowCollapsiblePanel should render collapsible panel row 1`] = `
+<VirtualizedList(RowCollapsiblePanel)
```

Comment:
Your snapshot isn't meaningful. It prints your component with its props, but that doesn't test your component.